### PR TITLE
[daint] Add DDT 21.0.2

### PR DESCRIPTION
**Pull request by kraushm**

### CODE REVIEW

**Comment by kraushm** [comment]
retest this please

File: easybuild/easyconfigs/d/ddt/ddt-21.0.2-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# jg (CSCS)
```
[the remark this answers is not in the document]
Comment (jgphpc) [reply easybuild/easyconfigs/d/ddt/ddt-21.0.2-linux-x86_64.eb:1]: no

**Comment by lucamar** [comment]
The build failed since the branch `kraushm:DDT/21.0.2` is way behind master, in particular regarding the custom easyblocks:
```
 easybuild/easyblocks/boost.py                                                       |    2 +-
 easybuild/easyblocks/bzip2cce.py                                                    |   10 +-
 easybuild/easyblocks/cpetoolchain.py                                                |    6 +-
 easybuild/easyblocks/darshan.py                                                     |    6 +-
 easybuild/easyblocks/esmfcray.py                                                    |    1 -
 easybuild/easyblocks/hpcg3.py                                                       |    6 +-
 easybuild/easyblocks/idl.py                                                         |    8 +-
 easybuild/easyblocks/julia.py                                                       |    5 -
 easybuild/easyblocks/juliabundle.py                                                 |   15 +-
 easybuild/easyblocks/juliapackage.py                                                |   24 +-
 easybuild/easyblocks/libqglviewer.py                                                |    5 +-
 easybuild/easyblocks/netcdf4_python.py                                              |    4 +-
 easybuild/easyblocks/netcdf_python.py                                               |    1 -
 easybuild/easyblocks/openfoamdev.py                                                 |    6 +-
 easybuild/easyblocks/pgi.py                                                         |    2 +-
 easybuild/easyblocks/pycuda.py                                                      |   11 +-
 easybuild/easyblocks/qt.py                                                          |   12 +-
 easybuild/easyblocks/rmpi.py                                                        |    5 +-
 easybuild/easyblocks/scons.py                                                       |    4 +-
 easybuild/easyblocks/xalt.py                                                        |    1 -
```

**Comment by kraushm** [comment]
Need to fix out-of-sync state of branch first.

**Pull request closed without merging**